searchresults.html

```diff
@@ -1,64 +1,77 @@
+                <div class="ResultsList_ArticleContainer">
+                    <h2 class="ResultsList_ArticleTitle">Search Title</h2>
+                    <p class="ResultsList_ArticleDescription">Lorem ipsum Lorem ipsum Lorem ipsum...</p>
+                    <button class="ResultsList_ArticleButton" type="button" onclick="alert('Hello world!')">Learn more</button>
                 </div>
-            </div>
-            <div class="search-result">
-                <img src="img/logo.png" alt="Search image" width="200px" height="200px">
-                <div class="search-container">
-                    <h4>Search Title</h4>
-                    <p>Lorem ipsum Lorem ipsum Lorem ipsum...</p>
-                    <button type="button" onclick="alert('Hello world!')">Learn more</button>
+            </article>
+            <article class="ResultsList_Article">
+                <img class="ResultsList_ArticleImage" src="img/logo.png" alt="Search image" width="200px" height="200px">
+                <div class="ResultsList_ArticleContainer">
+                    <h2 class="ResultsList_ArticleTitle">Search Title</h2>
+                    <p class="ResultsList_ArticleDescription">Lorem ipsum Lorem ipsum Lorem ipsum...</p>
+                    <button class="ResultsList_ArticleButton" type="button" onclick="alert('Hello world!')">Learn more</button>
                 </div>
-            </div>           
+            </article>    
         </section>
         <!--This is the section that includes the pagination-->
-        <section id="pagination">
-            <div class="pagination">
-                <a href="#">&laquo;</a>
-                <a href="#">1</a>
-                <a class="active" href="#">2</a>
-                <a href="#">3</a>
-                <a href="#">4</a>
-                <a href="#">5</a>
-                <a href="#">6</a>
-                <a href="#">&raquo;</a>
-              </div>
+        <section id="pagination" class="PaginationSection">
+            <ul class="Pagination">
+                <li class="PaginationElement">
+                    <a href="#">&laquo;</a>
+                </li>
+                <li class="PaginationElement">
+                    <a href="#">1</a>
+                </li>  
+                <li class="PaginationElement">
+                    <a href="#">2</a>
+                </li>      
+                <li class="PaginationElement">
+                    <a href="#">3</a>
+                </li>
+                <li class="PaginationElement">
+                    <a href="#">4</a>
+                </li>      
+                <li class="PaginationElement">
+                    <a href="#">5</a>
+                </li>   
+                <li class="PaginationElement">
+                    <a href="#">6</a>
+                </li>   
+                <li class="PaginationElement">
+                    <a href="#">&raquo;</a>
+                </li>       
+            </ul>
         </section>
```

Comment:
Incearca sa pastrezi consistenta in denumirea claselor, fie le scrii pe toate cu litere  mici si cratima, fie le scrii pe toate PascaCase sau camelCase. Cum iti este mai usor